extensions/content-manager/admin/src/containers/EditView/index.js

```diff
@@ -0,0 +1,317 @@
+        <Container className="container-fluid">
+          <Header />
+          <div className="row" style={{ paddingTop: 3 }}>
+            <div className="col-md-12 col-lg-9" style={{ marginBottom: 13 }}>
+              {formattedContentTypeLayout.map((block, blockIndex) => {
+                if (isDynamicZone(block)) {
+                  const {
+                    0: {
+                      0: { name },
+                    },
+                  } = block;
+                  const { max, min } = getField(name);
+
+                  return <DynamicZone key={blockIndex} name={name} max={max} min={min} />;
+                }
+
+                return (
+                  <FormWrapper key={blockIndex}>
+                    {block.map((fieldsBlock, fieldsBlockIndex) => {
+                      return (
+                        <div className="row" key={fieldsBlockIndex}>
+                          {fieldsBlock.map(({ name, size }, fieldIndex) => {
+                            const isComponent = getFieldType(name) === 'component';
+
+                            if (isComponent) {
+                              const componentUid = getFieldComponentUid(name);
+                              const isRepeatable = get(getField(name), 'repeatable', false);
+                              const { max, min } = getField(name);
+
+                              const label = get(getFieldMetas(name), 'label', componentUid);
+
+                              return (
+                                <FieldComponent
+                                  key={componentUid}
+                                  componentUid={componentUid}
+                                  isRepeatable={isRepeatable}
+                                  label={label}
+                                  max={max}
+                                  min={min}
+                                  name={name}
+                                />
+                              );
+                            }
+
+                            return (
+                              <div className={`col-${size}`} key={name}>
+                                <Inputs
+                                  autoFocus={
+                                    blockIndex === 0 && fieldsBlockIndex === 0 && fieldIndex === 0
+                                  }
+                                  keys={name}
+                                  layout={currentContentTypeLayoutData}
+                                  name={name}
+                                  onChange={() => {}}
+                                />
+                              </div>
+                            );
+                          })}
+                        </div>
+                      );
+                    })}
+                  </FormWrapper>
+                );
+              })}
+              {isRelationViewEnabled && (
+                <p className={isStructureCorrect ? '' : 'text-danger'}>
```

Comment:
how do you present "validation in progress" state? User should be notify about action doing in background, and should be inform about it result, so in that case we should show also a positive result, maybe "text-success" class will be good?

Reply:
fixed